script/foreman-debug

```diff
@@ -19,6 +19,8 @@ This program can be used on Foreman instances, Smart Proxy instances or
 backend services separately.
 
 OPTIONS:
+  -E      Detailed output incl. host names or domains
+          (THIS OPTION ADDS MORE SENSITIVE DATA IN THE REPORT)
```

Comment:
This is also somewhat misleading, the flag should be more specific to say that it's including contents of the database.